browser/ui/webui/settings/brave_privacy_handler.cc

```diff
@@ -30,6 +30,10 @@ BravePrivacyHandler::BravePrivacyHandler() {
       kRemoteDebuggingEnabled,
       base::Bind(&BravePrivacyHandler::OnRemoteDebuggingEnabledChanged,
                  base::Unretained(this)));
+  local_state_change_registrar_.Add(
```

Comment:
How about adding p3a things under the `BRAVE_P3A_ENABLED` guard?

Reply:
Good idea! Fixing up now...

Reply:
Updated!